.github/workflows/renovate-config.yml

```diff
@@ -0,0 +1,19 @@
+name: renovate-config-validator
```

Comment:
Do we need this validator or is it simply a nice-to-have?

Reply:
If we have no ci, we may merge invalid config into main branch. At that time, renovate app doesn't works.
This ci run when config file changed only, so I think it is better to add it.